roboteam_ai/test/ConditionTests/BallInDefenseAreaAndStillTest.cpp

```diff
@@ -19,7 +19,7 @@ TEST(DetectsDefenseArea, BallInDefenseAreaAndStill)
     EXPECT_TRUE(nodeTheirDefenceArea.theirDefenceArea); // check if the property is handled properly
 
     BBpointer->setBool("theirDefenceArea", false);
-    rtt::ai::BallInDefenseAreaAndStill node("Test", BBpointer);
+    rtt::ai::BallInDefenseAreaAndStill node("BallInDefenseAreaAndStill", BBpointer);
```

Comment:
I personally think for tests we should make sure the nodes have an explicitly different name so there is no chance in hell they can ever get confused. (that's why it's different I think)